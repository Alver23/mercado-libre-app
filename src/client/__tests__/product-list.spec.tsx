// Dependencies
import { shallow, ShallowWrapper } from 'enzyme';

// Under test file
import ProductList from '@alversoft/pages/products';

describe('<ProductList />', () => {
  let component: ShallowWrapper;

  beforeEach(() => {
    component = shallow(<ProductList />);
  });

  it('should save a snapshot of the component', () => {
    expect(component).toMatchSnapshot();
  });
});
