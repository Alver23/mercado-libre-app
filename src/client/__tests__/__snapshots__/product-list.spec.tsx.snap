// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProductList /> should save a snapshot of the component 1`] = `
<Layout
  meta={
    <Meta
      title="Product List"
    />
  }
>
  <div>
    <p>
      Product list
    </p>
  </div>
</Layout>
`;
